plasma_framework/test/src/exits/payment/PaymentPiggybackInFlightExitOnOutput.test.js

```diff
@@ -118,17 +119,20 @@ contract('PaymentInFlightExitRouter', ([_, alice, inputOwner, outputOwner, nonOu
                     exitTarget: inputOwner,
                     token: ETH,
                     amount: 999,
+                    piggybackBondSize: PIGGYBACK_BOND,
```

Comment:
seeing your test, it seems like it should start with size 0 here if piggyback should change the size?
eh...but I guess it would be hard without proper coding the bond size now. Probably left some TODOs?

Reply:
you are right, I'll fix that in a sec